codesign/lib/src/google_cloud_storage.dart

```diff
@@ -14,27 +14,20 @@ class GoogleCloudStorage {
   GoogleCloudStorage({
     required this.processManager,
     required this.rootDirectory,
-    required this.gCloudDownloadPattern,
-    required this.gCloudUploadPattern,
+    required this.gCloudDownloadUploadPath,
```

Comment:
This should remain in the methods instead of the constructor.

If we ever need to support multiple downloads/uploads, it would have to be changed.

Reply:
Sure, refactored.